### PR TITLE
52348_Approve and delete orders

### PR DESCRIPTION
**Pull request by fonqhoanq**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52348)

## WHAT (optional)
- approve and delete orders

## HOW
- in order controller create destroy and update function.
- in view create list orders

## Evidence (Screenshot or Video)
Screenshot from 2022-08-23 10-26-07

### CODE REVIEW

**Review by diemntn-0165 — APPROVED** [review]

**Comment by fonqhoanq** [comment]
ready

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+
   def index
     @pagy, @orders = pagy Order.all.newest, items: Settings.pagy.item_5
   end
 
-  def destroy; end
+  def edit; end
+
+  def update
+    if @order.update status: 1
```
Comment (manhnt-0960) [inline app/controllers/admin/orders_controller.rb]:
```suggestion
    if @order.active!
```

File: app/views/admin/orders/_order.html.erb [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
   <th scope="row"><%= order.id %></th>
-  <td><%= order.user_id %></td>
   <td><%= order.user_name %></td>
+  <td><%= order.phone_number %></td>
   <td><%= order.address %></td>
-  <td><%= order.status %></td>
+  <td>
+    <% case %>
+    <% when order.inactive? %>
+    <span class="badge badge-secondary"><%= t(".inactive") %></span>
```
Comment (manhnt-0960) [inline app/views/admin/orders/_order.html.erb]: thụt vào 1  tab, đoạn này e có thể move vào helper cho clear

**Comment by fonqhoanq** [comment]
ready

**Pull request merged**